src/Simplex/Chat/Messages.hs

```diff
@@ -395,7 +395,7 @@ instance StrEncoding ACIFileStatus where
 -- to conveniently read file data from db
 data CIFileInfo = CIFileInfo
   { fileId :: Int64,
-    fileStatus :: ACIFileStatus,
+    fileStatus :: Maybe ACIFileStatus,
```

Comment:
this - not sure it needs to be Maybe here.

Reply:
it has information about direction.. I'd rather not supply default - we don't know where it might be used. As of now we could set some default in db based on existing usages - e.g. status as complete (still there's arbitrary direction then) so basically it doesn't try to cancel - but if some code tries to perform some action based on it I'd rather have it know that this information is unknown. The change in type barely affects code base, right it's only a couple of places, and it doesn't participate in api